custom/icds_reports/models/views.py

```diff
@@ -208,7 +208,7 @@ class AggLsMonthy(models.Model):
         2) agg_awc
         3) awc_location_months
     """
-    supervisor_id = models.TextField(blank=True, null=True)
+    supervisor_id = models.TextField(primary_key=True)
```

Comment:
i dont think you want this to be primary key. a view doesnt have a primay key

Reply:
There is not need but. As discussed. Without this django keeps using id column. This is just for the prevention of that behaviour